.github/workflows/auto-approve.yml

```diff
@@ -1,10 +1,17 @@
 name: Auto approve
-on: pull_request
+on:
+  workflow_run:
+    workflows: ["test"]
+    types:
+      - completed
 
 jobs:
   build:
     runs-on: ubuntu-20.04
+    if: >
```

Comment:
One question here: What does the > do?

Reply:
It allows me to write the line in multi-line format (for readability) but is interpreted as one single line. The formal name is called formal style. 

So this
```yaml
  if: >
      github.event.workflow_run.event == 'pull_request' &&
      github.event.workflow_run.conclusion == 'success'
```

Becomes this

```yaml
  if: "github.event.workflow_run.event == 'pull_request' && github.event.workflow_run.conclusion == 'success'"
```

There are multiple multi-line styles for YAML that have different purposes. But I don't remember them off the top of my head lol. So I always refer to this website https://yaml-multiline.info/